{"name": "test_user_login", "status": "passed", "attachments": [{"name": "log", "source": "40c03649-dd88-4516-a0c1-94ca44f5f6f5-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "e9f5bb0d-2568-42de-aeb0-f1c1a9a0421a-attachment.txt", "type": "text/plain"}], "start": 1755437797700, "stop": 1755437798727, "uuid": "f30a3eea-5129-43fa-8e0e-dfebd1b60f13", "historyId": "a62cb8f188df29483727faa8d036c975", "testCaseId": "a62cb8f188df29483727faa8d036c975", "fullName": "selenium_tests.pytest_pages.test_user_login_invalid#test_user_login", "labels": [{"name": "parentSuite", "value": "selenium_tests.pytest_pages"}, {"name": "suite", "value": "test_user_login_invalid"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "6316-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "selenium_tests.pytest_pages.test_user_login_invalid"}], "titlePath": ["selenium_tests", "pytest_pages", "test_user_login_invalid.py"]}